### ISSUE TITLE
Flag 539 -- Possible issue with 7.7.4 and 7.2.3

### ISSUE THREAD

**Issue by 2008zjqc** [abtassociates/LSASampleCode#628]
Good morning Molly,
We have a flag 539: 
'The   SystemPath field in LSAExit must include PSH or be equal to all other   combinations or not applicable (i.e., any of the following SystemPath values:   -1, 8, 9, 10, 11, or 12) if the household exited from PSH (as reported in the   ExitFrom field). The following row in your LSAExit file violates that rule.'

Our test case is:
One person enrolled into PSH program from 10/28/2011 -- 11/02/2016 without Housing moving date  with exit dest 3
also this person enrolled into ES program from 10/1/2012 -- 11/11/2012 with exit dest 99

With the above case:
In 7.7.2--7.7.3 It grabs  PSH application which with exit dest 3, then set ExitFrom=8 due to moving date is NULL.

Due to the PSH.entrydate is prior to 10/1/2012, the last Inactive Date is set as '9/30/2012'.

In 7.7.4
Due to 'psh.EntryDate > ex.LastInactive' this condition, the PSH.EntryDate was set as NULL.
Thus the summary is set to be '1'. 

Therefore, in LSA_Calculated, we have one record with Exit from=8, but PSH records didn't make in as it been excluded from 7.7.4, as summary=1, only ES made it in. Above all are the reasons the flag was triggered. 

In summary, we think when generate basic records in Exit table, it didn't take into consideration of 10/1/2012, but in 7.7.4 it does has such constraint. Please advice on how to handle this. Thanks very much.

Best,
Chen

**Comment by MollyMcEvilley** [comment]
Thank you!  Updating the joins in 7.7.2-7.7.4 so that **SystemPath** includes enrollments with entry dates prior to 10/1/2012 when **LastInactive** = 9/30/2012 and the enrollments extend beyond that date

![image](https://user-images.githubusercontent.com/6156815/101028920-e6237b80-3546-11eb-9b40-44d9385049c9.png)
